client_opts.go

```diff
@@ -55,6 +56,14 @@ func WithDefaultRuntime(rt string) ClientOpt {
 	}
 }
 
+// WithDefaultPlatform sets the default platform matcher on the client
+func WithDefaultPlatform(platform platforms.MatchComparer) ClientOpt {
```

Comment:
`WithPlatform` by definition when you overwrite this its no longer the `default` right?

Reply:
I think it is like the `DefaultRuntime`. The default runtime is `runc`, but we can overwrite it with a different "default runtime".

Reply:
Ok no prob. I can see it both ways here :)